htdocs/core/actions_massactions.inc.php

```diff
@@ -1335,8 +1335,8 @@
 		// Message for elements well deleted
 		if ($nbok > 1) {
 			setEventMessages($langs->trans("RecordsDeleted", $nbok), null, 'mesgs');
-		} elseif (!empty($nbok)) {
-			setEventMessages($langs->trans("RecordsDeleted", '1'), null, 'mesgs');
+		} elseif ($nbok == 1) {
+			setEventMessages($langs->trans("RecordDeleted"), null, 'mesgs');
 		}
```

Comment:
@eldy, thank you for merge.

But i used "Record**s**Deleted" trans for only 1 element too voluntarily.
Because if we get 2 messages, one warning for linked object, and one for success deletions, i think it's more clear to get something like :

![image](https://user-images.githubusercontent.com/5585819/132688051-a4d478ce-19cd-4007-9d94-63cf680175a4.png)

than something like that : 

![image](https://user-images.githubusercontent.com/5585819/132688119-521e2b3a-19b6-4a72-bb05-097ade8c1105.png)

What's your opinion about that ?